tests/test_query.py

```diff
@@ -373,15 +373,15 @@ def test_incorrect_pssm_order():
 
     # check that error is thrown for incorrect pssm
     with pytest.raises(ValueError):
-        _ = q.build(conservation)
+        _ = q.build([conservation])
```

Comment:
I'd rather add a check in the build method in which if the module is not a list (because it's only one) then we put it in a list, similarly to what we do for many other parameters.

Reply:
OK. done in 1dd44efc